app/models/poll/voter.rb

```diff
@@ -12,7 +12,6 @@ class Voter < ActiveRecord::Base
 
     validates :poll_id, presence: true
     validates :user_id, presence: true
-    validates :token, presence: true
```

Comment:
🤔  why is it not needed anymore? Is it running us into problems?

No Poll::Voter from web origin will be ever valid without it. I would add this back, as well as a database constraint to mark that column as not nullable (empty token would still be storable but at least it covers some parts)

Reply:
Maybe I'm wrong but it may have relation with the fact that the poll_voter factory wasn't generating a token value and thus some specs failed?

I fixed it on upstream with https://github.com/AyuntamientoMadrid/consul/pull/956/commits/79f050f4286b22399a37171b2faf0e432323afdf